### DISCUSSION TITLE
Changes in custom Partials not recognized by tailwindcss ... --watch

### DISCUSSION THREAD

**Discussion by cladjidane** [tailpress/tailpress#67] [Q&A] [answered]
Hello, I would like to use Tailpress with the principle of cutting by file that Sass allows (partials). I would have liked for example to be able to add files in tailpress/resources/css/app.css in this way: 

```
@import "tailwindcss/base";
@import "tailwindcss/components";
@import "tailwindcss/utilities";

/* SASS APP */
@import "sprite", "cards";
```

It works without problems but the watcher doesn't look at these files and so I have to restart the compilation by hand. I looked for tailwind, but it seems that the tailwind watcher only takes one input file (-i, -input file), so I guess it doesn't see the others.

In previous versions of Tailpress running under mix, it worked very well.

Do you have a solution for this problem?

Thanks for your answer and for this great project!

**Comment by jeffreyvr** [comment, marked as answer]
@cladjidane I'm thinking of switching back to Mix for the next release as a workaround was found for an issue I had in combination with Tailwind 2.2.0.

In the mean time, you can use this branch (up to date version of TailPress with Mix): https://github.com/jeffreyvr/tailpress/tree/mix

**Reply by cladjidane** [reply]
I'm going to do it that way in this case :) Thanks!
